app/src/main/java/com/mashup/lastgarden/utils/Util.kt

```diff
@@ -0,0 +1,36 @@
+package com.mashup.lastgarden.utils
+
+import android.content.Context
+import android.content.res.Resources
+import com.mashup.lastgarden.R
+import java.text.DecimalFormat
+
+class Util {
+    companion object {
+        fun convertDate(resources: Resources, date: String): String {
```

Comment:
```suggestion
object Util {
        fun convertDate(resources: Resources, date: String): String {
```
어차피 전부 companion 이면 object로 만들어주세요. 그리고 Utils가 좋을 것 같아요

Reply:
아니면 StringFormatter? 로 만들어도 좋을 것 같네요. 결국 string을 format하는 기능들인것 같아서요

Reply:
저라면 StringFormatter를 만들어서 object로 만들거나 class로 만들어서 Application scope에서 DI 해줄 것 같아요.

Reply:
찾아봤는데 어떤걸 말하는지 잘 모르겠어요..😢  이름을 StringFormatter로 바꾸고 DI를 하라는걸까요..?!

Reply:
음 네 좀 어려우면 그냥 StringFormatter object class로 만들어주세요

Reply:
넵!!